src/sqlparser/src/parser.rs

```diff
@@ -2473,6 +2473,20 @@ impl Parser {
         }
     }
 
+    pub fn parse_returning(
```

Comment:
Could you please also add some unit tests for this, including parse and unparse?